airflow/providers/amazon/aws/utils/rds.py

```diff
@@ -22,5 +22,5 @@
 class RdsDbType(Enum):
     """Only available types for the RDS."""
 
-    INSTANCE: str = "instance"
-    CLUSTER: str = "cluster"
```

Comment:
What was the reason for removing this? Not a blocker, just curious

Reply:
ah yes, it's because this annotation is wrong. When you access `RdsDbType.INSTANCE` you don't get a string, you get an enum type, and the string is in `RdsDbType.INSTANCE.value`.

Reply:
I don't know, enums are weird, I don't even know what type to put here.
Also, it's an enum, IDEs already deduce the correct type without annotations.